docs/docs/cmd/flow/flow-get.mdx

```diff
@@ -279,6 +279,7 @@ m365 flow get --environmentName Default-d87a7535-dd31-4437-bfe1-95340acd55c5 --n
         },
         "outputs": {}
       },
+      "plan": "NotSpecified",
```

Comment:
It's upwards of 1500 lines to add the entire response here. I'm still thinking though that we'd better do it. Let me check with the others.

Reply:
Well it depends on the size of the flow. I've had 15K+ lines in a flow with four actions so... I don't really know if it's useful.

Reply:
I just had one with 1600 lines, so 1.6 K. 15.000 lines is a bit much indeed 😀

Reply:
It depends hugely on the trigger, if I try a flow with a manual trigger it's about 250 lines, the one I used was a File created or modified (SP), which was about 15k lines.

Reply:
Ok, awesome! So let's use one with a manual trigger 😂